### ISSUE TITLE
PropertySpacing considers PHP 8 attribute definition as blank lines

### ISSUE THREAD

**Issue by juagarc4** [slevomat/coding-standard#1572]
I defined class properties using PHP 8 attributes and I have configured "SlevomatCodingStandard.Classes.PropertySpacing" in this way:

```
<rule ref="SlevomatCodingStandard.Classes.PropertySpacing">
      <properties>
          <property name="minLinesCountBeforeWithComment" value="1" />
          <property name="maxLinesCountBeforeWithComment" value="1" />
          <property name="minLinesCountBeforeWithoutComment" value="1" />
          <property name="maxLinesCountBeforeWithoutComment" value="1" />
    </properties>
</rule>
```

An small snippet of the class is:

```php
    #[ORM\Column(name: 'position', type: 'integer', nullable: false)]
    public int $position;   <= (Line 34)

    #[ORM\ManyToOne(targetEntity: ClassName::class, inversedBy: 'stepItems')]
    #[ORM\JoinColumn(nullable: false)]
    public int $num;
```

An the message is:
```
34 | ERROR | [x] Expected 1 blank line after property, found 3. (SlevomatCodingStandard.Classes.PropertySpacing.IncorrectCountOfBlankLinesAfterProperty)
```

As you can see the two lines defining the property are considered as blank lines:
```php
    #[ORM\ManyToOne(targetEntity: ClassName::class, inversedBy: 'stepItems')]
    #[ORM\JoinColumn(nullable: false)]
```

Versions of the tools:
- PHP: 8.1.18
- Symfony 5.4
- squizlabs/php_codesniffer  3.7.1 
- slevomat/coding-standard: 6.4.1

Did I forget to configure anything else or is it really a bug?

**Comment by kukulich** [comment]
Please use latest version of Slevomat CS.
